server/providers/uploads/s3/index.ts

```diff
@@ -33,7 +35,20 @@ export default class AwsProvider implements Provider {
       credentials: {
         accessKeyId: options.credentials.accessKeyId,
         secretAccessKey: options.credentials.secretAccessKey
-      }
+      },
+      requestHandler: new NodeHttpHandler({
+        httpsAgent: new Agent({
+          maxSockets: 500,
+
+          // keepAlive is a default from AWS SDK. We want to preserve this for
+          // performance reasons.
+          keepAlive: true,
+          keepAliveMsecs: 1000,
+          timeout: 5000
```

Comment:
Maybe bump this up a bit in case the file is very large? Although at some point I do want to just rewrite rCDS to upload files directly (or at least properly stream files with the server as a middleman instead of keeping the whole thing in memory).

Reply:
Done - I'll go with what people claimed was the old timeout of 120000 ms